docs/general-development/sharepoint-design-manager-device-channels.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: SharePoint Design Manager device channels
-ms.date: 09/25/2017
+description: Describes provides an overview of device channels and describes how to plan for and configure a device channel in SharePoint.

```

Comment:
Let's take off "Describes" at the beginning of this sentence or delete "provides."